README.md

```diff
@@ -1,34 +1,76 @@
-#Children of Ur Web Client#
+# `coUclient`
+
+> Children of Ur's Dart-based browser client
 
-##What is this?##
 This repository contains the source code for Children of Ur's Dart-based browser client.
-The project is currently hosted at <a href="http://childrenofur.com" target="_blank">childrenofur.com</a>.
 
-Children of Ur is based on Tiny Speck's browser-based game, Glitch™. The original game's elements have been released into the public domain.
+[![Codeship Status for ChildrenOfUr/coUclient](https://codeship.com/projects/7e85d760-15e5-0132-d849-622a88ccaa2e/status?branch=master)](https://codeship.com/projects/33763)
+
+## License
+
+Children of Ur is based on Tiny Speck's browser-based game, Glitch&trade;. The original game's elements have been released into the public domain.
 For more information on the original game and its licensing information, visit <a href="http://www.glitchthegame.com" target="_blank">glitchthegame.com</a>.
 
-##Getting Started##
-1. Download the <a href="https://www.dartlang.org/">Dart Editor</a>
-2. In the Dart Editor, go to File -> "Open Existing Folder" and open this project folder
-3. Make sure you have the required dependencies specified in pubspec.yaml. If you're missing
-any of these, try selecting a file in the project, and then running Tools > Pub Get.
+License information for other assets used in Children of Ur can be found in `ATTRIBUTION.md`.
+
+## Usage
+
+The code is live at <a href="http://childrenofur.com" target="_blank">childrenofur.com</a>.
+
+If you want to run it locally or on your own server, you'll need to have an environment with [Dart](https://www.dartlang.org/) installed. Note that this repository does not currently contain any prebuilt files, so you'll also need a development environment. See [Contributing](#contributing) below.
+
+## Contributing
+
+`coUclient` is based on [Dart](https://www.dartlang.org/), so the first thing you'll need to do (if you haven't already) is to install it.
+
+### Setting up a development environment
+
+#### Mac OS X / macOS via `homebrew`
+
+1. `brew update`
+2. `brew tap dart-lang/dart`
+3. `brew install dart --with-content-shell --with-dartium`
+
+#### Windows
 
-###Test in Dartium###
-Right clicking on the `web/game.html` file and selecting 'Run in Dartium' will fire up the client.
+1. Download the <a href="https://www.dartlang.org/">Dart Editor</a>.
```

Comment:
I don't think the Dart Editor (a customization of Eclipse) is still supported. Might be better to suggest Atom or Sublime Text, or IntelliJ with the Dart plugin.